docs/docs/metrics.md

```diff
@@ -0,0 +1,150 @@
+apps.
+
+`service.mesosphere.marathon.group.count` (gauge) - The number of defined groups. In general,
+this is one number which influences the performance of Marathon. If you have
+a high number of groups, your performance will be lower than for a low number of
+groups. Note that each level between the slashes in your app IDs corresponds to
+a group. The app `/shop/frontend` is in the `frontend` group which is in
+the `shop` group which is in the root group.
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.task.running.count` (gauge) - The number of tasks that are
+currently running.
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.task.staged.count` (gauge) - The number of tasks that are
+currently staged. After tasks are launched, they are first in this state.
+A consistently high number of staged tasks indicates a lot of churn in Marathon
+and Mesos. Either you have many app updates/manual restarts or some of your apps
+have stability problems and are consistently automatically restarted.
+
+### Task update processing
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.core.task.update.impl.ThrottlingTaskStatusUpdateProcessor.queued` (gauge) -
+ number of queued status updates.
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.core.task.update.impl.ThrottlingTaskStatusUpdateProcessor.processing` (gauge) -
+ number of currently processing status updates.
+ 
+ <span class="label label-default">v0.15</span>
+ `service.mesosphere.marathon.core.task.update.impl.TaskStatusUpdateProcessorImpl.publishFuture` (timer) -
+ times processing status updates.
+
+### Configuration update processing
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.state.GroupManager.queued` (gauge) - number of queued app configuration
+updates. The maximum is configured with `--max_queued_root_group_updates`.
+
+<span class="label label-default">v0.15</span>
+`service.mesosphere.marathon.state.GroupManager.processing` (gauge) - number of currently processed app configuration
+updates. Since we serialize these updates, this is either 0 or 1.
+
+### Repositories
+
+Marathon stores its permanent state in "Repositories". These are the most important ones:
+
+* `GroupRepository` for app configurations and groups.
+* `TaskRepository` for the latest known task state. This is the repository with the largest
+  data churn.
+
+There are also the
+
+* AppRepository for versioned app configuration.
+* DeploymentRepository for currently running deployments.
+* TaskFailureRepository for the last failure for every application.
+* and more...
+
+For each of the repositories we have statistics about read and write requests. Substitute `*` with
+the repository of interest:
+ 
+`service.mesosphere.marathon.state.*.read-request-time.count` - the number of read requests.
```

Comment:
Do you mean prefixing the items with "* " in the markdown?

Actually, for me the dot for the list would only contribute useless clutter. I guess it is a matter of taste.

Have you looked at the rendered output?

If you feel strongly about this, I can change it.